internal/controller/monitoring.coreos.com/prometheusrule_controller.go

```diff
@@ -105,7 +105,7 @@ func (r *PrometheusRuleReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		err = utils.UpdateStatus(ctx, slo, r.Client, "Ready", metav1.ConditionFalse, "SLI Object not found")
 		if err != nil {
 			log.Error(err, "Failed to update SLO status")
-			//r.Recorder.Event(slo, "Error", "SLIObjectNotFound", "SLI Object not found")
+			r.Recorder.Event(slo, "Error", "SLIObjectNotFound", "SLI Object not found")
```

Comment:
As discussed already, "Error" is not a valid Event state

Reply:
Well, is it better now?